deploy/helm/jumpstarter/charts/jumpstarter-controller/templates/controller-deployment.yaml

```diff
@@ -51,10 +51,20 @@ spec:
           - -metrics-bind-address=:8080
         env:
         - name: GRPC_ENDPOINT
-          {{ if .Values.hostname }}
-          value: gprcs://{{ .Values.hostname }}
+          {{ if .Values.grpc.endpoint }}
```

Comment:
those are to let the controller know which endpoints to announce to the clients :D